extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/accessibilityControls.js

```diff
@@ -0,0 +1,74 @@
+var CONSTANTS = require('./../constants/constants');
+
+var AccessibilityControls = function (controller) {
+    this.controller = controller;
+    this.state = {
+      "fastForwardRate": 1,
+      "lastKeyDownTime": null
+    };
+    document.addEventListener("keydown", this.handleKey.bind(this));
+};
+
+AccessibilityControls.prototype = {
+  handleKey: function(e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 14 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)